packages/css-framework/src/components/_transfer.scss

```diff
@@ -0,0 +1,59 @@
+.rn-transfer {
+  .column-left,
+  .column-right {
+    width: 40%;
+    float: left;
```

Comment:
flexbox?

Reply:
Float works perfectly well and is supported further back in time. I see no benefit to using flexbox here.

Reply:
We have used Flexbox elsewhere I think, I'll let our designer css guru make the call :)

Reply:
Cool. I'll go with the flow.